version.txt

```diff
@@ -1 +1 @@
-3.1.0-beta.1
+4.0.0-beta.2
```

Comment:
I bumped this file so that builds coming from the `fennec/beta` branch will have a more up-to-date version https://github.com/mozilla-mobile/android-components/issues/5262